app/helpers/tabbed_nav_helper.rb

```diff
@@ -29,4 +28,15 @@ def edit_nav_item(label, href, current)
       },
     ]
   end
+
+  def current_tab
+    current_tab_name = request.path.split("/") & items
```

Comment:
Maybe this one can be `tab_name` (though really it's an array, so should be something like `potential_tab_names`?

Reply:
have just kept it as current tab and returning the first element of the array itself.